CONTRIBUTING.md

```diff
@@ -153,66 +180,127 @@ Sometimes, it might not make sense for ECS to standardize a field, value, or cat
+
+1. To begin making a new rule, you will use the `create-rule` sub-command to walkthrough guided steps to populate the content of a new toml rule file (that matches your target behavior).
+
+1. For example, to create a new rule file for `rules/windows/defense_evasion_msbuild_child.toml`, run the command:
+
+```shell
+    python -m detection_rules create-rule rules/windows/defense_evasion_msbuild_child.toml
+```
+
+1. You will be prompted with a series of entries to build the content of your new rule. It will iterate through the required metadata, and prompt for each field. The command will prompt you for each required metadata field:  
 
-The command will prompt you for each required field in the metadata
 ```
-Rule type (machine_learning, query, saved_id): query
+Rule type (machine_learning, saved_query, query, threshold, eql) (machine_learning, saved_query, query, threshold, eql):
+rule_id [86af1430-c0cd-428c-96e3-964268645120] ("n/a" to leave blank)  (required):
 actions (multi, comma separated):
-description (required): Look for child processes of MsBuild
+description (required):
 enabled [false] ("n/a" to leave blank):
 from [now-6m] ("n/a" to leave blank):
 false_positives (multi, comma separated):
 filters (multi, comma separated):
 interval [5m] ("n/a" to leave blank):
-exceptions_list (multi, comma separated):
 max_signals [100] ("n/a" to leave blank):
 meta:
-name (required): Suspicious Child of MsBuild
+name (required):
 note:
 references (multi, comma separated):
 risk_score [21] ("n/a" to leave blank)  (required):
-rule_id [90d0c543-e197-46d8-934d-0320b2c83486] ("n/a" to leave blank)  (required):
-severity [low] ("n/a" to leave blank)  (required): medium
-tags (multi, comma separated): Windows  
+severity [low] ("n/a" to leave blank)  (required):
+tags (multi, comma separated):
 throttle:
 timeline_id:
 timeline_title:
 to [now] ("n/a" to leave blank):
-threat (multi, comma separated):
-index (multi, comma separated): winlogbeat-*
-language [kuery] ("n/a" to leave blank)  (required): kuery
-query (required): event.category:process and process.parent.name:msbuild.exe
-ecs_version (multi, comma separated): 1.4.0
+add mitre tactic? [y/N]:
+index (multi, comma separated):
+language [kuery] ("n/a" to leave blank)  (required):
+query (required):
+author (required)  (multi, comma separated):
+building_block_type:
+exceptions_list (multi, comma separated):
+license [Elastic License] ("n/a" to leave blank)  (required):
+risk_score_mapping (multi, comma separated):
+rule_name_override:
+severity_mapping (multi, comma separated):
+timestamp_override: N
+mitre tactic name (required):
+technique IDs for Command and Control (required)  (multi, comma separated):
+add mitre tactic? [y/N]:
 ```
 
-Pending no errors, you'll see this output upon success
+1. You will receive this output when successful:
+
 ```
-Rule Suspicious Child of MsBuild saved to rules/windows/defense_evasion_msbuild_child.toml
+Rule saved to /PATH/TO/detection-rules/RULE.toml
```

Comment:
I think it's okay to be inexact in the output. Also, you don't know what path the user specified, it might not be in the `detection-rules` folder at all.
```suggestion
Rule saved to <path>.toml
```